src/custom/Feedback/style.tsx

```diff
@@ -151,7 +150,7 @@ export const FeedbackSubmitButton = styled(Button)<SubmitProp>(({ isOpen }) => (
 }));
 
 export const FeedbackButton = styled(Button)<RenderPositionType>(({ theme, renderPosition }) => ({
-  backgroundColor: theme.palette.mode === 'dark' ? BUTTON_MODAL_DARK : BUTTON_MODAL,
+  backgroundColor: BUTTON_MODAL,
```

Comment:
@captain-Akshay, is there any merit to leaving the support for dark mode and light mode in here, but simply having the same color for both?

Reply:
@nebula-aac, I assume that, generally speaking, we prefer to have all uses of colors instrumented for light and dark mode support even if there is no change - just a general principle to follow.

Reply:
since it was not visible with dark theme in meshmap with the side bar like we discussed. @leecalcote 
and the modal header is fixed already